ui/test/e2e/.eslintrc.cjs

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const cwd = process.cwd();
+const directories = cwd.split(path.sep);
+directories.pop();
+const updatedPath = directories.join(path.sep);
+const synpressPath = path.join(updatedPath, '/node_modules/@agoric/synpress');
+
+module.exports = {
```

Comment:
Because in `offer-up dApp` we are using ES modules. But `@agoric/synpress` uses commonJS modules. 

On line 24:
https://github.com/agoric-labs/synpress/blob/dev/package.json